pyvrp/cpp/search/LocalSearch.cpp

```diff
@@ -396,7 +431,8 @@ LocalSearch::LocalSearch(ProblemData const &data, Neighbours neighbours)
       orderRoutes(data.numVehicles()),
       lastModified(data.numVehicles(), -1),
       clients(data.numClients() + 1),
-      routes(data.numVehicles(), data),
+      routes(),
```

Comment:
This is not necessary; the default constructor is called when `routes()` isn't in the initialiser list here.

Reply:
Thanks